release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1883,6 +1883,15 @@ In the following tables, features are marked with the following statuses:
 
 * The DNF package manager included in {op-system-first} images cannot be used at runtime, because DNF relies on additional packages to access entitled nodes in a cluster that are under a Red Hat subscription. As a workaround, use the `rpm-ostree` command instead. (link:https://issues.redhat.com/browse/OCPBUGS-35247[*OCPBUGS-35247*])
 
+* When installing a cluster on {azure-full}, the installation will fail if no `install-config.yaml` file is provided or if the `controlPlane.platform.azure` stanza is null in the provided `install-config.yaml` file. See xref:../installing/installing_azure/ipi/installing-azure-customizations.html#installation-azure-config-yaml_installing-azure-customizations[Sample customized install-config.yaml file for Azure] for a sample configuration file, or set a non-null parameter as in the following example:
```

Comment:
The link should be an 'href' for an HTML file or a .adoc file for and 'xref', yes? It did resolve though.

Ascii writer's guide: https://asciidoctor.org/docs/asciidoc-writers-guide/#cross-references

Reply:
Good catch, I switched it to .adoc